calico_containers/pycalico/ipam.py

```diff
@@ -474,11 +629,11 @@ def _auto_assign(self, ip_version, num, handle_id,
                           host, pool)
```

Comment:
https://github.com/projectcalico/libcalico/pull/74/files#diff-17a9750fd8a984de4deef40d0202fa26R600

This should mention the difference in behavior when strict affinity is enabled.